src/xcode/ENA/ENA/Source/Services/Exposure Submission/ExposureSubmissionError.swift

```diff
@@ -5,102 +5,32 @@
 import Foundation
 
 enum ExposureSubmissionError: Error, Equatable {
-	case other(String)
-	case noRegistrationToken
+	case unsupported
+	case `internal`
+	case rateLimited
 	case enNotEnabled
 	case notAuthorized
-	case coronaTestServiceError(CoronaTestServiceError)
-
-	/// User has not granted access to their keys
-	case keysNotShared
-
-	/// Access to keys was granted but no keys were collected by the exposure notification framework
-	case noKeysCollected
-
-	case noSubmissionConsent
```

Comment:
where are all these removed errors gone?

Reply:
noSubmissionConsent is now part of ExposureSubmissionServicePreconditionError

Reply:
In general: some of them where not used anymore. A lot of them where moved to Resources but not deleted here.